pages/index.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+	<main>
+		<navbar />
+
+		<h1>Photography Lounge</h1>
+
+		<p>
+			Discord Partnered Community striving to create a welcoming and inclusive
+			space for Photographers to share their work, get critiqued, and have a fun
+			time!
+		</p>
+
+		<div id="icon-container">
+			<a
+				href="https://twitter.com/photographydisc"
+				target="_self"
+				rel="noreferrer"
+			>
+				<i class="fab fa-twitter fa-lg"></i>
+			</a>
+
+			<a href="https://discord.gg/photography" target="_self" rel="noreferrer">
+				<i class="fab fa-discord fa-lg"></i>
+			</a>
+
+			<a
+				href="https://reddit.com/r/photographerslounge"
+				target="_self"
+				rel="noreferrer"
+			>
+				<i class="fab fa-reddit fa-lg"></i>
+			</a>
+
+			<a href="https://vagabondit.surge.sh" target="_self" rel="noreferrer">
+				<i class="fas fa-crown fa-lg"></i>
+			</a>
+		</div>
+	</main>
+</template>
+
+<style lang="scss">
```

Comment:
Needs scope or why are you overwriting it here? If its meant to be global then add to `main.css`

Reply:
Added scope